### PR TITLE
docs: update sass customization text and examples

### PR DESCRIPTION
**Pull request by nekosaur**
## Description
update sass customization text and examples

## Motivation and Context
The current text and examples are incorrect

## How Has This Been Tested?
tried out the changes on both the sample project in https://github.com/vuetifyjs/vuetify/issues/7977 and on a clean vue-cli project with vuetify plugin.

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/lang/en/customization/SassVariables.json [hunk 1]
```diff
@@ -10,5 +10,7 @@
   "sassAlert2": "If you are requiring the Vuetify stylesheet by a `<link>` tag in your index file, remove it.",
   "sassText5": "Under the hood, the new variables will be hoisted to the top of the compiled styles and used to override the default behavior.",
   "sfcHeader": "## Single-file components",
-  "sfcText1": "When vue-cli compiles styles, by default it does not associate different file types for SASS, (**.scss** and **.sass**). When adding custom styles in a SFC (single-file component) with `lang=\"sass\"`, it will throw an error saying that a _semi-colon_ must be added to the import statement. When you add this _semi-colon_, it will tell you to remove it. This is due to the fact that Vuetify uses both **.scss** and **.sass** files. In order to resolve this, we have to tell webpack to explicitly use the **sass-loader** for **.sass** files."
+  "sfcText1": "Using the above configuration you will also be able to access Vuetify variables in SFC files when using style blocks with `lang=\"sass\"`. However if you wish to use `lang=\"scss\"`, an extra step is necessary.",
```
Comment (KaelWD) [inline packages/docs/src/lang/en/customization/SassVariables.json]: Don't we have scss files too?

Comment (MajesticPotatoe) [reply packages/docs/src/lang/en/customization/SassVariables.json]: Isnt that what its trying to say is that the config will work if using `sass` and if you want to uses `scss`, there some extra work involved? `However if you wish to use lang=\"scss\", an extra step is necessary."` ?

Comment (KaelWD) [reply packages/docs/src/lang/en/customization/SassVariables.json]: We have scss files, which should be affected by that. @nekosaur did you test this with a bunch of components actually loaded, or just an empty project?

Comment (nekosaur) [reply packages/docs/src/lang/en/customization/SassVariables.json]: I tested by using the default template that comes with vue-cli vuetify plugin, and added a few random components. Things seemed to work just fine.

**Pull request merged**